reconcile/change_owners/change_owners.py

```diff
@@ -306,74 +311,75 @@ def run(
         #   D E C I S I O N S
         #
 
-        gl = init_gitlab(gitlab_project_id)
-        approver_decisions = get_approver_decisions_from_mr_comments(
-            gl.get_merge_request_comments(
-                gitlab_merge_request_id, include_description=True
+        with init_gitlab(gitlab_project_id) as gl:
```

Comment:
til that the GitlabApi object also a context manager. i assume is supports some cleanup?
are you aware what exactly it is cleaning up?

Reply:
I added it as part of https://github.com/app-sre/qontract-reconcile/pull/3370/files#diff-1d5ab1abb6ecb1d8c243a9e8c04d4cf6505e6be2c8932df654ea9a11c6c1b64b, just close session connections.

Reply:
nice!